docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.8'
+services:
+ dynamodb-local:
+   command: "-jar DynamoDBLocal.jar -sharedDb -dbPath ./data"
+   image: "amazon/dynamodb-local:2.0.0"
+   container_name: dynamodb-local
+   ports:
+     - "8000:8000"
+   volumes:
+     - "./docker/dynamodb:/home/dynamodblocal/data"
```

Comment:
It's already ignored by https://github.com/mbta/api/blob/ff123f84665f0f1bb91c61a02496c27dc000eadb/.gitignore#L38. I've switched it to use `inMemory` anyway since I don't think we need the file.